app/src/main/res/values/strings.xml

```diff
@@ -1,14 +1,149 @@
+    <string name ="activity_profile_icon_delete_account">Delete account</string>
+
+    <!-- activity maps -->
+
+    <!-- activity main unregister -->
+
+    <!-- activity main -->
+
+    <!-- activity login -->
+    <string name ="activity_login_title">Login to your account</string>
+    <string name ="activity_login_email_hint">Email</string>
+    <string name ="activity_login_password_hint">Password</string>
+    <string name="activity_login_login_button">Login</string>
+    <string name="activity_login_create_account_button">New Here? Create Account</string>
+
+    <!-- activity list -->
+    <string name="activity_list_title">Activities</string>
+
+    <!-- GPS recording -->
+    <string name="gps_recording_start">Start</string>
+
+    <!-- activity edit profile -->
+    <string name="edit_profile_title">User Settings</string>
+    <string name="edit_profile_picture">Profile Picture</string>
+    <string name="edit_profile_full_name">Your Name</string>
+    <string name="edit_profile_email">Your Email Address</string>
+    <string name="edit_profile_phone">Your Phone</string>
+    <string name="edit_profile_description">Description</string>
+    <string name="edit_profile_save_profile_data">Save profile</string>
+
+    <!-- activity description -->
+    <string name="activity_description_title">Title</string>
+    <string name="activity_description_by">By </string>
+    <string name="activity_description_with"> with </string>
+    <string name="activity_description_during"> during </string>
+    <string name="activity_description_hours"> hours </string>
+    <string name="activity_description_Organisator">Organisator</string>
+    <string name="activity_description_what">What ?</string>
+    <string name="activity_description_when">When ?</string>
+    <string name="activity_description_where">Where ?</string>
+    <string name="activity_description_sport">Sport</string>
+    <string name="activity_description_number_participant">Number participant</string>
+    <string name="activity_description_participants">participants</string>
+    <string name="activity_description_date">date</string>
+    <string name="activity_description_duration">Duration</string>
+    <string name="activity_description_address">Address</string>
+    <string name="activity_description_description">Address</string>
+    <string name="activity_description_register_button">Register</string>
+    <string name="activity_description_how_to_go_button">How to Go ?</string>
+    <string name="activity_description_chat_button">Chat</string>
+    <string name="activity_description_sign_in">Log in to register</string>
+
+    <!-- chat -->
+    <string name="activity_chat_addMessageImageView">addMessageImageView</string>
+    <string name="activity_chat_type_post">Type your post</string>
+    <string name="activity_chat_initial_message">Initial chat welcome message</string>
+    <string name="activity_send_image_button">Send Image</string>
+
+
```

Comment:
👏🤩 Thanks for this amazing work!

Reply:
Haha no worries! We had to do it at some point to follow the android conventions. Everyone put his stone to the building so that the app is cool! 😉

Reply:
As you say yes!